pack/mut_gen.json

```diff
@@ -1030,6 +1049,7 @@
 		"resource_wild": 1,
 		"set_code": "brawler",
 		"set_position": 3,
+		"text": "<b>Hero Interrupt</b> <i>(attack/defense)</i>: When an enemy attacks, spend 1 resource of any type \u2192 prevent 3 damages from this attack and deal 3 damage to that enemy. Remove this card from the game and the campaign pool.",
```

Comment:
I think this should say `prevent 3 damage from` instead of `prevent 3 damages from`